.github/workflows/locale_update_all.yaml

```diff
@@ -0,0 +1,60 @@
+name: "Locale Update All"
+
+on:
+  schedule:
+  - cron: '0 0 1,15 * *'
```

Comment:
I'm not sure what schedule this should be done but I was thinking twice per month is probably ok.  I still need to do the work to run `rake locale:po_to_json` with the result of this PR on manageiq-ui-classic but that can be done later.